### ISSUE TITLE
Add documentation for supported data types

### ISSUE THREAD

**Issue by nwstephens** [rstudio/db.rstudio.com#80]
Add a table that lists [supported data types](https://github.com/r-dbi/odbc/blob/master/R/DataTypes.R
) for each data source. 

See <https://github.com/r-dbi/odbc/issues/238>

**Comment by edgararuiz** [comment]
This feels like it should be a vignette in the actual package
@jimhester - Do you agree?

**Comment by jimhester** [comment]
Yeah maybe a vignette with the table and also how to define a mapping manually or for databases which are not included in the package.
